File: official/docs/python/v8/referral-customers/add-payment-method-with-bank-account.py
import easypost
import os

client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))

payment_method = client.beta_referral_customer.add_payment_method(
    stripe_customer_id="cus_...",
    payment_method_reference="ba_...",
    primary_or_secondary="primary",
)

print(payment_method)

File: official/docs/ruby/v6/referral-customers/add-payment-method-with-bank-account.rb
require 'easypost'

client = EasyPost::Client.new(api_key: 'EASYPOST_API_KEY')

payment_method = client.beta_referral_customer.add_payment_method(
  'cus_...',
  'ba_...',
  'primary',
)

puts payment_method
